automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -22,7 +22,17 @@ abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAl
   var claimedBillingProject: ClaimedProject = _
   var billingProject : GoogleProject = _
   var ronCluster: Cluster = _
-
+  def enableWelder: Boolean = true
```

Comment:
Nice, so existing tests will default to having welder enabled?

Reply:
Answering my own question: it seems like tests inheriting `ClusterFixtureSpec` will have welder enabled, but some tests use `LeonardoTestUtils.createNewCluster()` directly. I wonder if we should enable welder for those tests too. It would be good to make sure welder doesn't affect any other functionality..